.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,38 @@
+
+<!--
+Thank you for your interest in contributing to XCSoar! Please fill out the
+following questions to make it easier for us to review your changes.
+
+You do not need to check all the boxes below all at once, feel free to take
+your time and add more commits. If you're done and ready for review, please
+check the last box.
+-->
+
+
+What is the purpose of this pull request? What does it change?
+--------------------------------------------------------------
+
+<!--
+Please describe the changes here, as detailed as needed.
+-->
+
+
+Has the changes been discussed in an issue or the forum?
+--------------------------------------------------------
+
+<!--
+Please link issues and relevant forum posts here.
+
+If this PR resolves an issue on GitHub, write "Closes #1234" so that the issue
+is closed automatically when this PR is merged.
+-->
+
+
+Checklist
+---------
+
+- [ ] I have enabled [maintainer edits for this PR](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork)
+- [ ] I have added tests for all changes in this PR
+- [ ] I have added documentation for the changes (in the manual)
+- [ ] I've formatted commit messages in the same style as [other commits in the repo](https://github.com/XCSoar/XCSoar/commits/master)
```

Comment:
Do we need to point that out here? When the author submits his PR, his commits are already done and pushed. It's too late.

Reply:
Perhaps in the XCSoar project it works well, but in most other projects it's not very common that commit message style is followed. Just because the PR is filed it doesn't mean changes can't be made, so it's not too late. This item becomes a question of "do we want to remind people that they should format their commit messages properly?". I'm fine with either, it's up to you!

Reply:
Changed to a friendly reminder in the top comment.